tron/core/job_scheduler.py

```diff
@@ -49,11 +49,18 @@ def enable(self):
             return
 
         self.job.enabled = True
-        self.create_and_schedule_runs(ignore_last_run_time=True)
+        self.create_and_schedule_runs(next_run_time='current')
 
-    def create_and_schedule_runs(self, ignore_last_run_time=False):
-        for job_run in self.get_runs_to_schedule(ignore_last_run_time):
-            self._set_callback(job_run)
+    def create_and_schedule_runs(self, next_run_time=None):
```

Comment:
Feels like a bad practice to accept different types of arguments here: the string 'current', None (which means last run time), or a datetime... What if it just accepted a datetime or None? 

Debatable what makes sense as a default if the next_run_time is None, but I think "current" makes sense because that's the default if you pass in None to `self.job.scheduler.next_run_time` (in `get_runs_to_schedule`). 

Then you'd have to move the logic about getting the last_run_time to the `schedule` method.

Reply:
Done.